packages/mobile/src/App.tsx

```diff
@@ -79,21 +81,30 @@ const App = () => {
         <PortalProvider>
           <ToastContextProvider>
             <ErrorBoundary>
-              <NavigationContainer>
-                {!isReadyToSetupBackend ? (
-                  <WebAppAccountSync
-                    setIsReadyToSetupBackend={setIsReadyToSetupBackend}
-                  />
-                ) : null}
-                <Airplay />
-                <RootScreen isReadyToSetupBackend={isReadyToSetupBackend} />
-                <Drawers />
-                <Modals />
-                <Audio />
-                <OAuth />
-                <NotificationReminder />
-                <OfflineDownloader />
-              </NavigationContainer>
+              <WalletConnectProvider
```

Comment:
so do we need this wrapped in the root vs just around the wallet-connect screen?

Reply:
we don't! we can wrap further down!

Reply:
word word